benchmarks/LIA/2013.OOPSLA_Hola/find.sl

```diff
@@ -0,0 +1,33 @@
+#need to change this based on having a aux var that tracks if we found
+
```

Comment:
Please move this file to `others` subdirectory since it wasn't part of the original HOLA benchmarks.